assets/js/modules/adsense/components/dashboard/LegacyAdSenseDashboardWidgetSiteStats.js

```diff
@@ -161,8 +161,7 @@ export default function LegacyAdSenseDashboardWidgetSiteStats( props ) {
 			<Row>
 				<Cell size={ 12 }>
 					<GoogleChart
-						chartType="line"
-						selectedStats={ [ selectedStats ] }
+						chartType="LineChart"
```

Comment:
Same here, this chart stopped showing the previous period. See it here: https://google.github.io/site-kit-wp/storybook/pull/2916/?path=/story/adsense-module-components-module-overview-widget--loaded

Here is how it looks in the develop branch: https://google.github.io/site-kit-wp/storybook/develop/?path=/story/adsense-module-components-module-overview-widget--loaded